validator/lib/validateTileset.js

```diff
@@ -102,6 +108,12 @@ function validateTileHierarchy(root, tilesetDirectory) {
             }
         }
 
+        if(defined(tile.refine)) {
+            if (tile.refine !== 'ADD' && tile.refine !== 'REPLACE') {
+                return 'Refine property in tileset must have either ADD or REPLACE as its value.';
```

Comment:
Change `tileset` to `tile`. Also surround `ADD` and `REPLACE` without quotation marks.